server/athenian/api/controllers/miners/github/contributors.py

```diff
@@ -21,14 +21,42 @@ async def mine_contributors(repos: Collection[str],
                             time_to: Optional[datetime],
                             db: databases.Database,
                             cache: Optional[aiomcache.Client],
-                            with_stats: Optional[bool] = True) -> List[dict]:
+                            with_stats: Optional[bool] = True,
+                            with_roles: List[str] = None) -> List[dict]:
```

Comment:
The model has `as` while we use `with` here. We should stick to one variant of naming everywhere. How about `as_roles`